docs/user_manual/selecting.md

```diff
@@ -40,6 +40,13 @@ If you want to delete the entire selection, the easiest way is to deselect every
 ### Inverting Selection
 Inverting the selection means that all of the currently selected pixels will get deselected, while all of the unselected pixels will get selected. You can do this by either pressing <kbd>Control + I</kbd>, or going to the Select menu and pressing **Invert**. If you have everything selected, inverting will clear the selection. Likewise, if you have nothing selecting, inverting will select everything.
 
+## Duplicating
+If you **don't** have an active selection, holding <kbd>Control + Alt + Mouse click</kbd> will *duplicate* whatever was below it and place it in active selection.
```

Comment:
Control + Alt + Mouse will not have any effect if you don't have an active _selection_. I think you probably meant to write [active transformation](https://orama-interactive.github.io/Pixelorama-Docs/user_manual/transforming#having-an-active-transformation).